get-contributing.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+GHURL="https://raw.githubusercontent.com/expressjs/express/master"
+DEST="en/resources/contributing.md"
+
+# Remove everything after the <!-- marker -->
+sed -i "" '1,/<!-- marker -->/!d' $DEST
+
+curl -s "$GHURL/Contributing.md" | sed '1 i\
```

Comment:
Note to self, this first `sed` command could really just be an `echo` appending the line into the file. All that sed command is doing is adding inserting the "Note" about origin of the text before the content of the curl'd file.

Reply:
Even on that note, I think we likely don't need to actually link out to it from the website; I'm not sure it _really_ adds much. I think the links were mainly there as part of it being _manual_ in order to act as a notice for us / folks trying to update it. We could likely just leave HTML comments instead and let the doc just flow on the website naturally.